dartdoc_test/test/testdata/run analyze command.txt

```diff
@@ -0,0 +1,17 @@
+Extracting code samples ...
+Analyzing code samples ...
+No issues found!
+No issues found!
```

Comment:
I'm guessing we're writing such a line for each file without issues.

We could write: `example/lib/filename.dart: No issues found!`

But this seems like this is something that's only really worth writing in verbose mode.